charts/pega/charts/installer/config/prbootstrap.properties.tmpl

```diff
@@ -15,5 +15,9 @@ com.pega.pegarules.bootstrap.tempdir=/opt/pega/temp
 poolPreparedStatements=true
 install.{{ .Env.DB_TYPE }}.connectionProperties={{ .Env.JDBC_CUSTOM_CONNECTION }}
 install.{{ .Env.DB_TYPE }}.password={{ .Env.DB_PASSWORD }}
+{{- if isTrue .Env.BYPASS_LOAD_ENGINE_CLASSES }}
+com.pega.pegarules.bootstrap.codeset.version.Pega-EngineCode=do_not_set
```

Comment:
What behavior do we expect for "do_not_set" option? Does it some legacy logic for such value?

Reply:
If it is empty string, there were some validation failures that it cannot be empty.
Also, when it is loading classes from DB, it will load with provided code set version. If there is invalid value(like do_not_set), then it cannot load from Database. Subsequently it will load from the packed engine, which we want it.

Reply:
Can we do some other solution for it? Such [magic values](https://en.wikipedia.org/wiki/Magic_string) can create unnecessary complications in the code. 
I would suggest something like instead of it.
```
com.pega.pegarules.bootstrap.codeset.isCodeVersionSet = false
```

Reply:
I recommend using `FS_ONLY` to match the behaviour of the platform scripts.  It makes things much clearer in the logs.

Reply:
are you suggesting 
com.pega.pegarules.bootstrap.codeset.version.Pega-EngineCode=FS_ONLY?

I feel adding one more variable like com.pega.pegarules.bootstrap.codeset.isCodeVersionSet = false is redundant

Reply:
Yeah,
```
com.pega.pegarules.bootstrap.codeset.version.Pega-EngineCode=FS_ONLY
```
should work.  Trying to add a new property would almost definitely require some engine changes.

Reply:
@Max-Levitskiy There absolutely needs to be a value in the `com.pega.pegarules.bootstrap.codeset.version.Pega-EngineCode` property in order to suppress loading DB code with how the engine currently works.  Doesn't actually matter what the value is, just needs to not be a valid codeset version (and small enough to fit in the column due to database platform quirks).

Reply:
I have done changes as 

com.pega.pegarules.bootstrap.codeset.version.Pega-EngineCode=FS_ONLY.

Kindly approve.

Reply:
@rrajarap could I ask you to add some comment near this code explaining why do we set this property? I think it could help people who will modify it later. :)

Reply:
Added those comments.